### PR TITLE
hosted: Update version for Mender client 4.0.4 release

### PR DESCRIPTION
**Pull request by lluiscampos**
Similar to #2423.

### CODE REVIEW

File: 08.Server-integration/03.Mutual-TLS-authentication/01.Keys-and-certificates/docs.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ export DOCKER_REGISTRY_USERNAME=
 export DOCKER_REGISTRY_PASSWORD=
 
 # The name and version of the mtls server container
-export MTLS_IMAGE=registry.mender.io/mendersoftware/mtls-ambassador:mender-3.7.5
+export MTLS_IMAGE=registry.mender.io/mendersoftware/mtls-ambassador:mender-3.7.6
```
Comment (TheYoctoJester) [inline 08.Server-integration/03.Mutual-TLS-authentication/01.Keys-and-certificates/docs.md:49]: Is this intentionally included in this PR?

**Review by TheYoctoJester — APPROVED** [review]
Pre-approved, with one minor comment to be clarified.

File: 08.Server-integration/03.Mutual-TLS-authentication/01.Keys-and-certificates/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 08.Server-integration/03.Mutual-TLS-authentication/01.Keys-and-certificates/docs.md:49]: Yes, same as in #2423.

As we are technically still using the "Mender bundle" release toolchain, all the backend services have been built and tagged even when where there are no source changes.

**Comment by lluiscampos** [comment]
> Pre-approved, with one minor comment to be clarified.

Thank you for the review :eyes:

**Pull request merged**